bundles/com.zeligsoft.domain.dds4ccm.ui/src/com/zeligsoft/domain/dds4ccm/ui/wizards/DDS4CCMModelWizardPage.java

```diff
@@ -120,7 +136,10 @@ protected void createModelTypeGroup(Composite composite){
 	} 
 	
 	public ModelTypeEnum getTargetModelType(){
-		
+		if(atcdSelectionButton == null){
```

Comment:
Why test the consequence of no AXCIOMA support rather than the looking at the primary source. Call your isAxciomaSupport() method instead.